it dhp - first line - p tag

kokoye2007 opened the pull request:
The first line of the verse HTML code is missing a <p> tag.

ayya-vimala commented:
Hi @kokoye2007  - apologies this has taken so long! Thanks for your changes. I agree with the additional <p> tags and would like to merge that. But the bilara-data you also add in the PR should not be there. Bilara-data are added to the bilara-data repo. Would you be willing to remove it here and then make a new PR in bilara-data on a new branch? Thanks so much!

kokoye2007 committed:
it dhp - Italian Dhammapada HTML: first line and sc274 tag fixes

- Add first line <p> tags across all 25 DHP chapters
- Fix sc274 tag formatting
